src/Core__JSON.resi

```diff
@@ -0,0 +1,55 @@
+type t = Js.Json.t
+
+type jsonReviver
+external asJsonReviver: 'a => jsonReviver = "%identity"
+type jsonReplacer
+external asJsonReplacer: 'a => jsonReplacer = "%identity"
```

Comment:
What's the point of these? They look unsafe, but are not marked as such. And if they can really be anything, why not just allow anything to be passed directly?

Also, as I understand, `jsonReviver` can really only be a function. So why isn't it a function?